src/rpc/blockchain.cpp

```diff
@@ -1110,7 +1110,7 @@ static RPCHelpMan verifychain()
                     {"nblocks", RPCArg::Type::NUM, RPCArg::DefaultHint{strprintf("%d, 0=all", DEFAULT_CHECKBLOCKS)}, "The number of blocks to check."},
                 },
                 RPCResult{
-                    RPCResult::Type::BOOL, "", "Verified or not"},
+                    RPCResult::Type::BOOL, "", "Verification finished successfully. If false, check debug.log for reason."},
```

Comment:
That would break the existing API though - there might be users out there who'd need to adjust automated scripts if -`verifychain` returned a more complicated JSON object instead of a bool. 
I think I'd ACK a pull that does that, but would prefer not to include it here unless people really want it.